pinot-controller/src/test/resources/recommenderInput/RealtimeProvisioningInput_timeColumn.json

```diff
@@ -135,7 +135,7 @@
   },
   "qps": 150,
   "numMessagesPerSecInKafkaTopic":1000,
-  "numRecordsPerPush":10000,
+  "numRecordsPerPush":10000000,
```

Comment:
RT provisioning rule is no longer using NRPP right

Reply:
It's not being used. I just deleted the line.